src/main/scala/com/microsoft/partnercatalyst/fortis/spark/sinks/cassandra/aggregators/ConjunctiveTopicsAggregator.scala

```diff
@@ -49,10 +49,10 @@ class ConjunctiveTopicsAggregator extends FortisAggregatorBase with Serializable
   }
 
   private def AllSourcesAggregateViewQuery: String = {
-    val GroupedColumns = GroupedBaseColumnNames ++ Seq("externalsourceid")
+    val GroupedColumns = GroupedBaseColumnNames ++ Seq("pipelinekey")
     val SelectClause = GroupedColumns.mkString(",")
 
-    s"SELECT $SelectClause, 'all' as pipelinekey, sum(mentioncount) as mentioncountagg " +
+    s"SELECT $SelectClause, 'all' as externalsourceid, sum(mentioncount) as mentioncountagg " +
```

Comment:
The 'all' in this case should have been for the `externalsourceid` field, not the `pipelinekey`.